src/modules/entities/modules/reports/services/reducers/entities/list.js

```diff
@@ -3,14 +3,14 @@ import { types } from '../../actions';
 const initialState = [];
 
 export default function listReportsReducer(state = initialState, action) {
+    const { params, meta } = action;
```

Comment:
this is not the best solution - because it will be triggered so many times - it should be as a part of those cases 
try to console.log here to see how many times these consts are re-declared